### PR TITLE
Add more jobs to Jenkins AWS

### PR DESCRIPTION
**Pull request by surminus**
In starting to think about how 2ndline might start to use the environment, add a few more jobs that might be used, and see if they work in the way we expect.

### CODE REVIEW

File: hieradata_aws/class/jenkins.yaml [hunk 1]
```diff
@@ -54,10 +54,18 @@ govuk_jenkins::config::banner_string: 'AWS Deploy'
 
 govuk_jenkins::job_builder::jobs:
   - govuk_jenkins::job::deploy_app
+  - govuk_jenkins::job::deploy_emergency_banner
   - govuk_jenkins::job::deploy_puppet
+  - govuk_jenkins::job::deploy_router_data
```
Comment (afda16) [inline hieradata_aws/class/jenkins.yaml]: Do we need to change the PLEK values of the job to work on AWS?

File: hieradata_aws/class/jenkins.yaml [hunk 2]
```diff
@@ -54,10 +54,18 @@ govuk_jenkins::config::banner_string: 'AWS Deploy'
...
   - govuk_jenkins::job::passive_checks
+  - govuk_jenkins::job::remove_emergency_banner
+  - govuk_jenkins::job::run_rake_task
+  - govuk_jenkins::job::search_benchmark
```
Comment (afda16) [inline hieradata_aws/class/jenkins.yaml]: In the configuration it uses: www-origin.integration.publishing.service.gov.uk, do we want to change this for AWS?

File: hieradata_aws/class/jenkins.yaml [hunk 3]
```diff
@@ -54,10 +54,18 @@ govuk_jenkins::config::banner_string: 'AWS Deploy'
...
+  - govuk_jenkins::job::search_index_checks
+  - govuk_jenkins::job::search_test_spelling_suggestions
```
Comment (afda16) [inline hieradata_aws/class/jenkins.yaml]: It uses www-origin.integration.publishing.service.gov.uk

File: hieradata_aws/class/jenkins.yaml (hunk 2 above)
Comment (surminus) [reply hieradata_aws/class/jenkins.yaml]: Probably :) I just blindly added some jobs, good stuff in actually doing in the hard work and checking them!

File: hieradata_aws/class/jenkins.yaml (hunk 1 above)
Comment (surminus) [reply hieradata_aws/class/jenkins.yaml]: This job is rarely used, so I'm going to remove for now.

**Comment by surminus** [comment]
Updated to address all comments.

**Pull request merged**